docs/tutorial/part-two/index.md

```diff
@@ -488,4 +488,5 @@ yet for your favorite CSS option,
 
 ## What's coming next?
 
-Now continue on to [Part Three](/tutorial/part-three/) of the tutorial.
+Now continue on to [Creating nested layout components
```

Comment:
```suggestion
Now continue on to [Creating nested layout components](/tutorial/part-three/), the third part of the tutorial.
```

Weird line break, right? But yeah, this does seem more consistent! I'd almost propose a combination though, e.g. `[Part Three - Creating Nested Layout Components]`

Reply:
Or something like in [part three](https://www.gatsbyjs.org/tutorial/part-three/)

Reply:
Hm, yeah these aren't consistent. The goal is to make sure people know what the next part of the tutorial covers, be friendly and conversational, and also to give them a doc title name they can recognize in the future (for example, if we use different versions of the title all over the place, beginners might start to doubt they even found the correct section of the tutorial).

I'd actually vote for Dustin's first suggestion and second suggestion `Now continue on to [Part Three - Creating Nested Layout Components] to learn more about creating sections of your site that you want to share across multiple pages` since it keeps the title consistent, gives a meaningful sneak peak into the next part of the tutorial, and is conversational.